ingesters/winevent.md

```diff
@@ -4,7 +4,7 @@ The Gravwell Windows events ingester runs as a service on a Windows machine and
 
 ## Basic Configuration
 
-The Windows Event ingester uses the unified global configuration block described in the [ingester section](#!ingesters/ingesters.md#Global_Configuration_Parameters).  Like most other Gravwell ingesters, the Windows Evenet ingester supports multiple upstream indexers, TLS, cleartext, and named pipe connections, a local cache, and local logging.
+The Windows Event ingester uses the unified global configuration block described in the [ingester section](#!ingesters/ingesters.md#Global_Configuration_Parameters).  Like most other Gravwell ingesters, the Windows Event ingester supports multiple upstream indexers, TLS, clear text, and named pipe connections, a local cache, and local logging.
```

Comment:
I think you should add "cleartext" to your spellcheck library. It's well-established in crypto/networking circles.